pkg/plugins/internal/machinery/scaffold_test.go

```diff
@@ -466,18 +409,6 @@ var _ = Describe("Scaffold", func() {
 	})
 })
 
-var _ model.Plugin = fakePlugin{}
```

Comment:
why we are removing the plugins?

Reply:
This is part of the old experimental plugin system used for addon.